### PR TITLE
Latest and greatest for web

### LINKED ISSUE #94: Fix URL wrapping for longer snapshot links
For long city names, the snapshot URLs split onto two lines in a way that might be confusing to novice users

Easy fix is to increase width of box or otherwise ensure that reasonable city names will fit on one line. Or, decrease line spacing, add a link, etc., with the goal of making the URL look like one thing.

Screen Shot 2020-04-07 at 8 19 53 PM

### LINKED ISSUE #95: Tweak text on data selection buttons
Can we change the button text to `PDI only`, `Add cases`, and `Add deaths`? This will clarify what each option does.

Also, can we only show the data source info when the relevant button is clicked?

When cases is clicked:
> Number of cases are for all of New York County and are updated daily
> Source: https://coronavirus.jhu.edu/map.html

When deaths is clicked:
> Number of deaths are for all of New York County and are updated daily
> Source: https://coronavirus.jhu.edu/map.html

Screen Shot 2020-04-07 at 8 21 16 PM

### PR DESCRIPTION
**Pull request by benjaminpkane**
* multi-graph
* miami
* Resolves #95 
* Resolves #94

### CODE REVIEW

**Review by lethosor — APPROVED** [review]
looks good!

File: web/src/components/chart.js [hunk 1]
```diff
@@ -36,6 +43,25 @@ import {
...
+const plotOptions = {
+  pdi: {
+    name: "PDI",
+    abbr: "PDI only",
+    primary: true,
+    color: "rgb(255, 109, 4)",
+  },
+  cases: {
+    name: "Number of cases",
+    abbr: "Add cases",
```
Comment (lethosor) [inline web/src/components/chart.js:55]: Just noticed - these are used primarily in the tooltips, so the tooltips now look like this:

image

Comment (benjaminpkane) [reply web/src/components/chart.js:55]: Right. Fixing!

Comment (lethosor) [reply web/src/components/chart.js:55]: Where did this fix end up? I'm seeing it in prod but not on develop

Comment (benjaminpkane) [reply web/src/components/chart.js:55]: https://github.com/voxel51/pandemic51/pull/97

**Pull request merged**